packs/data/feats.db/marshal-dedication.json

```diff
@@ -39,13 +39,42 @@
                     }
                 ],
                 "key": "Aura",
+                "predicate": {
+                    "not": [
+                        "marshalStance"
+                    ]
+                },
                 "radius": 10,
                 "slug": "marshals-aura",
                 "traits": [
                     "emotion",
                     "mental",
                     "visual"
                 ]
+            },
+            {
+                "effects": [
+                    {
+                        "affects": "allies",
+                        "events": [
+                            "enter"
+                        ],
+                        "uuid": "Compendium.pf2e.feat-effects.Ru4BNABCZ0hUbX7S"
+                    }
+                ],
+                "key": "Aura",
+                "predicate": {
+                    "all": [
+                        "marshalStance"
+                    ]
+                },
+                "radius": "{item|flags.pf2e.rulesSelections.marshallAuraSize}",
+                "slug": "marshals-stance",
+                "traits": [
+                    "emotion",
+                    "mental",
+                    "visual"
+                ]
             }
```

Comment:
@stwlam I'd like your input here. Adding one of the new effects above properly turns off the 10 foot aura RE above this one, but they don't turn this aura RE on.

If a Critical Success occurs using either stance, the whole aura should increase to 20 feet, so I'm trying to have this new RE turn on and set itself to the size of the stance, but it won't do it. Probably a timing issue?